hydra-node/src/Hydra/API/Server.hs

```diff
@@ -169,13 +174,18 @@ runAPIServer host port party tracer history callback responseChannel = do
   httpApp _ respond =
     respond $ responseLBS status400 [] "only WebSocket connections supported"
 
-  forwardGreetingOnly con = do
+  -- NOTE: We will add a 'Greetings' message on each API server start. This is
+  -- important to make sure the latest configured 'party' is reaching the
+  -- client.
+  forwardGreetingOnly con noHistory = do
+    hist <- STM.atomically (readTVar history)
+    let seqNo = bool (fromIntegral $ List.length hist) 0 noHistory
     time <- getCurrentTime
     sendTextData con $
```

Comment:
We are not using `sendOutput` here. Hence, the change to `sendOutput` above should not be necessary.